src/fu-engine.c

```diff
@@ -4880,6 +4881,10 @@ fu_engine_plugin_check_supported_cb (FuPlugin *plugin, const gchar *guid, FuEngi
 {
 	g_autoptr(XbNode) n = NULL;
 	g_autofree gchar *xpath = NULL;
+
+	if (!fu_config_get_only_supported (self->config))
```

Comment:
I think this breaks the "blacklisting" of https://github.com/fwupd/fwupd/blob/master/plugins/dfu/dfu.quirk#L15 too -- although you could argue the dfu quirk is kinda abusing the feature.

Reply:
Well that particular one it doesn't actually cause a failure to enumerate.  I think we could just as easily drop that quirk.  It's not like anyone will be able to do any damage to it.

Reply:
Fair point.